token-lending/program/src/math/decimal.rs

```diff
@@ -40,14 +40,13 @@ impl Decimal {
         Self(U192::zero())
     }
 
-    // OPTIMIZE: use const slice when fixed in BPF toolchain
     fn wad() -> U192 {
-        U192::from(WAD)
+        const DWAD: U192 = U192([1000000000000000000, 0, 0]);
```

Comment:
nit: what about underscoring these for legibility?

Reply:
ah yes good catch